decidim-meetings/config/locales/en.yml

```diff
@@ -22,7 +22,7 @@ en:
         end_time: End time
         id: ID
         iframe_access_level: Iframe access level
-        iframe_embed_type: Iframe embed type
+        iframe_embed_type: Only a few services allow the "Embed in meeting page" or "Open in live event page". Note that with the "Embed in the meeting page" option on mobiles, as the screen inherits different dimensions, it will actually work as an "Open in live event page".
```

Comment:
```suggestion
        iframe_embed_type: Iframe embed type
        iframe_embed_type_html: Only a few services allow the "Embed in meeting page" or "Open in live event page". Note that with the "Embed in the meeting page" option on mobiles, as the screen inherits different dimensions, it will actually work as an "Open in live event page".
```